tox.ini

```diff
@@ -2,13 +2,15 @@
 # fakeroot -u tox --recreate
 
 [tox]
-envlist = py32, py33, py34, py35
+envlist = py{32,33,34,35}{,-attic}
```

Comment:
right, this was at the suggestion of @RonnyPfannschmidt, as mentionned earlier. i didn't want to hardcode attic as a build dependency... but maybe it could just be added to the tox test suite?